src/index.js

```diff
@@ -0,0 +1,36 @@
+import './styles.css';
+import display from './modules/display.js';
+import { getAPI, setAPI, createGame } from './modules/API.js';
+
+const initializeGame = async () => {
+  console.log(await createGame('roblox'));
+
```

Comment:
- [ ] It would be great if you remove the console.log in order to not reveal the private information about your app to the end user 😨 and also to follow JavaScript best practices 👍